notify_python35.cpp

```diff
@@ -136,20 +136,12 @@ bool NotifyPython35::configure()
 				   m_pythonScript.c_str(),
 				   filterMethod.c_str());
 
-#if 0
 	// 2) Import Python script
-	// check first method name (substring of scriptname) has name SCRIPT_NAME
-	if (filterMethod.compare(SCRIPT_NAME) != 0)
+	// check first method name is empty:
+	// disable delivery, cleanup and return true
+	// This allows reconfiguration
+        if (filterMethod.empty())
 	{
```

Comment:
May be add a log message here indicating plugin has been disabled.

Reply:
There is already a comment in an early stage about scripName/filterMethod.

The code here silently allows re-configuration if the plugin is setup without a script: i.e the GUI step by step notification delivery setup